### PR TITLE
[many ports] Don't depend on default features of tiff

### PR DESCRIPTION
**Pull request by Thomas1664**
This removes the transitive dependency of liblzma via tiff

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by Osyotr** [comment]
Duplicate #38047?

**Comment by Thomas1664** [comment]
> Duplicate #38047?

Probably should have checked beforehand. But why stopping there?

**Comment by dg0yt** [comment]
> But why stopping there?

Because #38047 revises all dependencies in one port, instead of one dep in many ports. This reduces overlap with other (people's) PRs.

**Comment by Thomas1664** [comment]
> Because #38047 revises all dependencies in one port, instead of one dep in many ports. This reduces overlap with other (people's) PRs.

I will remove the changes to freeimage. However, I don't think it makes sense to open one PR per port because this would take like 30+ PRs.

**Review by ahzahzahz — COMMENTED** [review]
.\vcpkg.exe install opencv
Computing installation plan...
The following packages will be built and installed:
    opencv[core,default-features]:x64-windows@4.8.0#1
  * opencv4[core,default-features,dnn,jpeg,png,quirc,tiff,webp]:x64-windows@4.8.0#15
  * tiff[core,jpeg,lzma,zip]:x64-windows@4.6.0#4
  * vcpkg-get-python-packages:x64-windows@2024-01-24
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
-- Automatically setting %HTTP(S)_PROXY% environment variables to "127.0.0.1:1082".
Compiler found: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe
Restored 0 package(s) from C:\Users\112670\AppData\Local\vcpkg\archives in 312 us. Use --debug to see more details.
Installing 1/4 tiff[core,jpeg,lzma,zip]:x64-windows@4.6.0#4...
Building tiff[core,jpeg,lzma,zip]:x64-windows@4.6.0#4...
-- Using cached libtiff-libtiff-v4.6.0.tar.gz.
-- Cleaning sources at C:/vcpkg/buildtrees/tiff/src/v4.6.0-cea7694842.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/vcpkg/downloads/libtiff-libtiff-v4.6.0.tar.gz
-- Applying patch FindCMath.patch
-- Applying patch requires-lerc.patch
-- Using source at C:/vcpkg/buildtrees/tiff/src/v4.6.0-cea7694842.clean
-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: C:/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe -v
    Working Directory: C:/vcpkg/buildtrees/tiff/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\vcpkg\buildtrees\tiff\config-x64-windows-dbg-CMakeCache.txt.log
      C:\vcpkg\buildtrees\tiff\config-x64-windows-rel-CMakeCache.txt.log
      C:\vcpkg\buildtrees\tiff\config-x64-windows-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:252 (vcpkg_execute_required_process)
  ports/tiff/portfile.cmake:33 (vcpkg_cmake_configure)
  scripts/ports.cmake:175 (include)


error: building tiff:x64-windows failed with: BUILD_FAILED
Elapsed time to handle tiff:x64-windows: 13 s
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+tiff
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=[tiff]+Build+error+on+x64-windows&body=Copy+issue+body+from+C%3A%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md




error  log:
/////////////////////////////////////////////
[1/2] "C:/Program Files/CMake/bin/cmake.exe" -E chdir ".." "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/tiff/src/v4.6.0-cea7694842.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/tiff_x64-windows" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-Dcxx=OFF" "-Djpeg=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_JPEG=ON" "-Dlibdeflate=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_Deflate=OFF" "-Dlerc=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_LERC=OFF" "-Dlzma=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_liblzma=ON" "-Dtiff-tools=OFF" "-Dwebp=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_WebP=OFF" "-Dzlib=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_ZLIB=ON" "-Dzstd=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_ZSTD=OFF" "-DCMAKE_FIND_PACKAGE_PREFER_CONFIG=ON" "-Dtiff-docs=OFF" "-Dtiff-contrib=OFF" "-Dtiff-tests=OFF" "-Djbig=OFF" "-Djpeg12=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_OpenGL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_GLUT=ON" "-DZSTD_HAVE_DECOMPRESS_STREAM=ON" "-DHAVE_JPEGTURBO_DUAL_MODE_8_12=OFF" "-DCMAKE_MAKE_PROGRAM=C:/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../CMakeCache.txt 
"C:/Program Files/CMake/bin/cmake.exe" -E chdir ".." "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/tiff/src/v4.6.0-cea7694842.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/tiff_x64-windows" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-Dcxx=OFF" "-Djpeg=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_JPEG=ON" "-Dlibdeflate=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_Deflate=OFF" "-Dlerc=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_LERC=OFF" "-Dlzma=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_liblzma=ON" "-Dtiff-tools=OFF" "-Dwebp=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_WebP=OFF" "-Dzlib=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_ZLIB=ON" "-Dzstd=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_ZSTD=OFF" "-DCMAKE_FIND_PACKAGE_PREFER_CONFIG=ON" "-Dtiff-docs=OFF" "-Dtiff-contrib=OFF" "-Dtiff-tests=OFF" "-Djbig=OFF" "-Djpeg12=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_OpenGL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_GLUT=ON" "-DZSTD_HAVE_DECOMPRESS_STREAM=ON" "-DHAVE_JPEGTURBO_DUAL_MODE_8_12=OFF" "-DCMAKE_MAKE_PROGRAM=C:/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
-- Building tiff version 4.6.0
-- The C compiler identification is MSVC 19.39.33523.0
-- The CXX compiler identification is MSVC 19.39.33523.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test C_FLAG_W3
-- Performing Test C_FLAG_W3 - Success
-- Performing Test HAVE_LD_VERSION_SCRIPT
-- Performing Test HAVE_LD_VERSION_SCRIPT - Failed
-- Looking for assert.h
-- Looking for assert.h - found
-- Looking for fcntl.h
-- Looking for fcntl.h - found
-- Looking for io.h
-- Looking for io.h - found
-- Looking for strings.h
-- Looking for strings.h - not found
-- Looking for sys/time.h
-- Looking for sys/time.h - not found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for unistd.h
-- Looking for unistd.h - not found
-- Looking for getopt
-- Looking for getopt - not found
-- Looking for optarg
-- Looking for optarg - not found
-- Looking for mmap
-- Looking for mmap - not found
-- Looking for setmode
-- Looking for setmode - not found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of size_t
-- Check size of size_t - done
-- Found ZLIB: optimized;C:/vcpkg/installed/x64-windows/lib/zlib.lib;debug;C:/vcpkg/installed/x64-windows/debug/lib/zlibd.lib (found version "1.3.1")
-- Could NOT find Deflate (missing: Deflate_LIBRARY Deflate_INCLUDE_DIR) 
-- Found JPEG: optimized;C:/vcpkg/installed/x64-windows/lib/jpeg.lib;debug;C:/vcpkg/installed/x64-windows/debug/lib/jpeg.lib (found version "62")
-- Could NOT find JBIG (missing: JBIG_LIBRARY JBIG_INCLUDE_DIR) 
-- Could NOT find LERC (missing: LERC_LIBRARY LERC_INCLUDE_DIR) 
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - not found
-- Found Threads: TRUE
CMake Error at C:/vcpkg/installed/x64-windows/share/WebP/WebPConfig.cmake:15 (message):
  File or directory C:/vcpkg/installed/include referenced by variable
  WebP_INCLUDE_DIR does not exist !
Call Stack (most recent call first):
  C:/vcpkg/installed/x64-windows/share/WebP/WebPConfig.cmake:38 (set_and_check)
  C:/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  cmake/WebPCodec.cmake:29 (find_package)
  CMakeLists.txt:123 (include)


-- Configuring incomplete, errors occurred!
[2/2] "C:/Program Files/CMake/bin/cmake.exe" -E chdir "../../x64-windows-dbg" "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/tiff/src/v4.6.0-cea7694842.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/tiff_x64-windows/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-Dcxx=OFF" "-Djpeg=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_JPEG=ON" "-Dlibdeflate=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_Deflate=OFF" "-Dlerc=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_LERC=OFF" "-Dlzma=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_liblzma=ON" "-Dtiff-tools=OFF" "-Dwebp=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_WebP=OFF" "-Dzlib=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_ZLIB=ON" "-Dzstd=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_ZSTD=OFF" "-DCMAKE_FIND_PACKAGE_PREFER_CONFIG=ON" "-Dtiff-docs=OFF" "-Dtiff-contrib=OFF" "-Dtiff-tests=OFF" "-Djbig=OFF" "-Djpeg12=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_OpenGL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_GLUT=ON" "-DZSTD_HAVE_DECOMPRESS_STREAM=ON" "-DHAVE_JPEGTURBO_DUAL_MODE_8_12=OFF" "-DCMAKE_MAKE_PROGRAM=C:/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DEBUG_POSTFIX=d"
FAILED: ../../x64-windows-dbg/CMakeCache.txt 
"C:/Program Files/CMake/bin/cmake.exe" -E chdir "../../x64-windows-dbg" "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/tiff/src/v4.6.0-cea7694842.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/tiff_x64-windows/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-Dcxx=OFF" "-Djpeg=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_JPEG=ON" "-Dlibdeflate=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_Deflate=OFF" "-Dlerc=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_LERC=OFF" "-Dlzma=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_liblzma=ON" "-Dtiff-tools=OFF" "-Dwebp=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_WebP=OFF" "-Dzlib=ON" "-DCMAKE_REQUIRE_FIND_PACKAGE_ZLIB=ON" "-Dzstd=OFF" "-DCMAKE_REQUIRE_FIND_PACKAGE_ZSTD=OFF" "-DCMAKE_FIND_PACKAGE_PREFER_CONFIG=ON" "-Dtiff-docs=OFF" "-Dtiff-contrib=OFF" "-Dtiff-tests=OFF" "-Djbig=OFF" "-Djpeg12=OFF" "-DCMAKE_DISABLE_FIND_PACKAGE_OpenGL=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_GLUT=ON" "-DZSTD_HAVE_DECOMPRESS_STREAM=ON" "-DHAVE_JPEGTURBO_DUAL_MODE_8_12=OFF" "-DCMAKE_MAKE_PROGRAM=C:/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DEBUG_POSTFIX=d"
-- Building tiff version 4.6.0
-- The C compiler identification is MSVC 19.39.33523.0
-- The CXX compiler identification is MSVC 19.39.33523.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test C_FLAG_W3
-- Performing Test C_FLAG_W3 - Success
-- Performing Test HAVE_LD_VERSION_SCRIPT
-- Performing Test HAVE_LD_VERSION_SCRIPT - Failed
-- Looking for assert.h
-- Looking for assert.h - found
-- Looking for fcntl.h
-- Looking for fcntl.h - found
-- Looking for io.h
-- Looking for io.h - found
-- Looking for strings.h
-- Looking for strings.h - not found
-- Looking for sys/time.h
-- Looking for sys/time.h - not found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for unistd.h
-- Looking for unistd.h - not found
-- Looking for getopt
-- Looking for getopt - not found
-- Looking for optarg
-- Looking for optarg - not found
-- Looking for mmap
-- Looking for mmap - not found
-- Looking for setmode
-- Looking for setmode - not found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of size_t
-- Check size of size_t - done
-- Found ZLIB: optimized;C:/vcpkg/installed/x64-windows/lib/zlib.lib;debug;C:/vcpkg/installed/x64-windows/debug/lib/zlibd.lib (found version "1.3.1")
-- Could NOT find Deflate (missing: Deflate_LIBRARY Deflate_INCLUDE_DIR) 
-- Found JPEG: optimized;C:/vcpkg/installed/x64-windows/lib/jpeg.lib;debug;C:/vcpkg/installed/x64-windows/debug/lib/jpeg.lib (found version "62")
-- Could NOT find JBIG (missing: JBIG_LIBRARY JBIG_INCLUDE_DIR) 
-- Could NOT find LERC (missing: LERC_LIBRARY LERC_INCLUDE_DIR) 
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - not found
-- Found Threads: TRUE
CMake Error at C:/vcpkg/installed/x64-windows/share/WebP/WebPConfig.cmake:15 (message):
  File or directory C:/vcpkg/installed/include referenced by variable
  WebP_INCLUDE_DIR does not exist !
Call Stack (most recent call first):
  C:/vcpkg/installed/x64-windows/share/WebP/WebPConfig.cmake:38 (set_and_check)
  C:/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  cmake/WebPCodec.cmake:29 (find_package)
  CMakeLists.txt:123 (include)


-- Configuring incomplete, errors occurred!
ninja: build stopped: subcommand failed.

**Comment by Thomas1664** [comment]
@ahzahzahz I think this is actually an other issue and has nothing to do with liblzma. Therefore, it won't be fixed by this PR.

File: ports/colmap/portfile.cmake [hunk 1]
```diff
@@ -53,6 +49,8 @@ vcpkg_cmake_configure(
         -DGIT_COMMIT_ID=${GIT_COMMIT_ID}
         -DGIT_COMMIT_DATE=${COLMAP_GIT_COMMIT_DATE}
         -DOPENMP_ENABLED=${OPENMP_ENABLED}
+        -DCMAKE_FIND_PACKAGE_PREFER_CONFIG=ON
```
Comment (dg0yt) [inline ports/colmap/portfile.cmake:52]: IMO this is too invasive to be used for software with a wide and variable set of transitive dependencies. The trace will speak for one configuration (on each platform), but it doesn't study optional features.
And the changed mode may even break downstream usage of exported config.

**Comment by Thomas1664** [comment]
@WangWeiLin-MV this PR is ready for review.

**Comment by WangWeiLin-MV** [comment]
Please resolve conflicts.

**Review by WangWeiLin-MV — DISMISSED** [review]

**Review by data-queue — APPROVED** [review]

**Pull request merged**